sdk/storage/storage-blob/src/PageBlobRangeResponse.ts

```diff
@@ -1,23 +1,25 @@
 import { HttpResponse } from "@azure/core-http";
-import * as Models from "./generated/src/models";
+import { PageBlobGetPageRangesHeaders, PageBlobGetPageRangesDiffHeaders } from "./generatedModels";
+import {
+  PageBlobGetPageRangesResponse as PageBlobGetPageRangesResponseModel,
+  PageBlobGetPageRangesDiffResponse as PageBlobGetPageRangesDiffResponseModel
```

Comment:
You should be able to import `PageBlobGetPageRangesResponse` and `PageBlobGetPageRangesResponse` from  "./generatedModels"

Reply:
Thanks, I'll do that